composer.json

```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": "^7.1",
 
-        "sylius/sylius": "^1.0"
+        "sylius/sylius": "^1.1@dev"
```

Comment:
This should not be part of this PR. Could you rebase, please?

Reply:
It wasn't initially - @pamil added these.
Anyhow, to simplify things since this is just one-liner feel free to close this PR and just add that line yourselves.
Don't get me wrong - no problem if you still want me to do it (just let me know), but it seams more efficient if you guys just put that line in next release.

thx

Reply:
In fact, #204 added that (cause the change should affect 1.0 branch as well), thanks for the contribution anyway! :)